libs/react-helpers/src/lib/format/number.ts

```diff
@@ -2,6 +2,10 @@ import { BigNumber } from 'bignumber.js';
 import memoize from 'lodash/memoize';
 import { getUserLocale } from './utils';
 
+export function toDecimal(numberOfDecimals: number) {
+  return numberOfDecimals > 0 ? 1 / Math.pow(10, numberOfDecimals) : 1;
```

Comment:
nitpick (non-blocking): simplier
```suggestion
  return numberOfDecimals > 0 ? Math.pow(10, -numberOfDecimals) : 1;
```

Reply:
nice, thanks! actually realised the null check can also be omitted in both variations as `Math.pow(10, -0)` returns 1